tests/models/test_responses.py

```diff
@@ -22,17 +20,24 @@ async def async_streaming_body():
 
 
 def test_response():
-    response = httpx.Response(200, content=b"Hello, world!", request=REQUEST)
+    response = httpx.Response(
+        200,
+        content=b"Hello, world!",
+        request=httpx.Request("GET", "https://example.org"),
+    )
 
     assert response.status_code == 200
     assert response.reason_phrase == "OK"
     assert response.text == "Hello, world!"
-    assert response.request is REQUEST
+    assert response.request.method == "GET"
+    assert response.request.url == "https://example.org"
     assert response.elapsed >= datetime.timedelta(0)
     assert not response.is_error
 
 
 def test_raise_for_status():
+    REQUEST = httpx.Request("GET", "https://example.org")
```

Comment:
I guess let's use the same "use lowercase if not using globals" convention here, even if it results in a slightly larger diff since we have to change the rest of this test case?
```suggestion
    request = httpx.Request("GET", "https://example.org")
```